sql/updates/auth/3.3.5/2017_02_04_99_auth.sql

```diff
@@ -0,0 +1,15 @@
+DELETE FROM `rbac_permissions` WHERE `id` BETWEEN 846 AND 850;
+INSERT INTO `rbac_permissions` (`id`, `name`) VALUES
+(846, 'Command: npc spawngroup'),
+(847, 'Command: npc despawngroup'),
+(848, 'Command: gobject spawngroup'),
+(849, 'Command: gobject despawngroup'),
+(850, 'Command: list respawns');
```

Comment:
RBAC_PERM_COMMAND_GO_OFFSET already broke that sync. If I add the master ones in, one will be missing. I can't renumber offset as part of this PR, that would be silly. So, what is the proper procedure when this happens?